v2/event/event_unmarshal.go

```diff
@@ -200,7 +205,28 @@ func readJsonFromIterator(out *Event, iterator *jsoniter.Iterator) error {
 
 		// If no specversion, enqueue unconditionally
 		if !checkFlag(state, specVersionV03Flag|specVersionV1Flag) {
-			tokenQueue = append(tokenQueue, entry{key: key, value: iterator.ReadAny()})
+			switch key {
```

Comment:
Can you add a comment here specifying that those values can be parsed without the specversion?

Reply:
Yep!